docs/CONTRIBUTING.md

```diff
@@ -2,9 +2,13 @@
 
 **This document is under review** 
 
+### Join the Signalen community mailinglist to stay up to date
+
+* Product owners for each development team coordinate in the product steering group. If you wish to contribute or keep up to date please join the [email list](https://lists.publiccode.net/mailman/postorius/lists/signalen-discuss.lists.publiccode.net/) and/or [Slack channel](https://samenorganiseren.slack.com/archives/C01AF8CN495).
+
 ### Want to add a new feature?
 
-* **At the moment we do not accept merge request with new features from anyone other than the core developers!**
+* See [signalen.org](https://signalen.org/en/using-signalen) for how to help with development or propose new features.
```

Comment:
A small comment; if a open source developer built a useful feature we will have a look at the PR.

Reply:
That's certainly good to hear! :+1:  While I imagined that to be true, I didn't want to assume that would be the case, especially in light of the very strong language which this proposed change replaces.

While it's possible that someone will submit a "drive by" pull request, my expectation is that this will not happen very often, thus it's not a big gap at the moment. That said, if you'd like me to add something to cover that, I'm happy to do so.

Reply:
We will deal with it if and when it happens. Is good for now.